### ISSUE TITLE
Adversarial Loss for Cosmos Tokenizer

### ISSUE THREAD

**Issue by xiaoxiao26** [NVIDIA/Cosmos#133]
Hi, I'm looking to finetune the released Cosmos Tokenizer on my own data. The paper says "Additionally, we use adversarial loss in the fine-tuning stage to further enhance reconstruction details, particularly at large compression rates".

I don't see this loss implementation in the released post-training setup: https://github.com/NVIDIA/NeMo/blob/cosmos_tokenizer/nemo/collections/physicalai/tokenizer/losses/loss.py#L379-L381

Is the adversarial loss not needed, or is the implementation not released yet? Thank you!
